renderer/features/messages/AddMessage.js

```diff
@@ -30,83 +30,98 @@ const AddMessage = props => {
 
   const sendPaymentRequest = async () => {
     setMenuOpen(false);
-    const paymentRequestResponse = await queue.promptForm({
-      title: 'Request Payment',
-      body:
-        'Send a payment request by specifying how many sats you want them to pay and an optional memo.',
-      inputs: [
-        {
-          name: 'amount',
-          label: 'Amount',
-          required: true,
-          type: 'number'
-        },
-        {
-          name: 'memo',
-          label: 'Memo (Optional)'
-        }
-      ],
-      acceptLabel: 'Send',
-      cancelLabel: 'Cancel'
-    });
 
-    if (paymentRequestResponse) {
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
```

Comment:
thoughts on replacing this with a do/while loop instead? execute prompt and then while invalid answer is provided continue looping.  I don't feel that strongly against the while(true) but just curious to hear your thoughts on it.

Reply:
I don't have any strong opinion either way. do while seems more natural here since the flow must execute at least once, and we would also get rid of the eslint-disable, so I'll refactor in a new commit. Some people don't really like do while and I got more used to while (true) that's all.

Reply:
yeah, I tend to agree.  I never really see do/while being used and have seen while(true).  I guess let's try the do/while if you don't mind to get rid of the eslint-dsable?

Reply:
let me give it a try and see how it looks :-)

Reply:
try it and while(true) just works better in this case.

that's because you can't do ` const { amount, memo } = paymentRequestResponse;` if you don't have `paymentRequestResponse`, but not having `paymentRequestResponse` you cannot force having it either, so you can't remove that outside the loop.

Reply:
ah okay, that makes sense.  Let's just leave the while(true) then.

Reply:
yeah, I think is better. I thought also about removing the duplication, but decided to leave as this for the time being, I think at some point it might make sense to also check for the balance before sending, or the maximum amount that can be received